src/components/CardItem/CardItem.js

```diff
@@ -14,10 +14,13 @@ const CardItem = ({ card }) => (
   </Card>
 );
 
+const cardType = PropTypes.shape({
+  body: PropTypes.string,
```

Comment:
Unfortunately, this can't be done from the front end, this is how we receive the JSON object. The back end has to change its structure and then as a consumer we can change it.

Reply:
Ok. cool. got it.